providers/inmemory/shedlock-provider-inmemory/src/main/java/net/javacrumbs/shedlock/provider/inmemory/InMemoryLockProvider.java

```diff
@@ -36,7 +39,7 @@
         }
     }
 
-    boolean isLocked(String lockName) {
+    public boolean isLocked(String lockName) {
```

Comment:
What is the reason for making this public?

Reply:
different package for `InMemoryLockProviderIntegrationTest` - there is a cycle between support module and `InMemoryLockProvider`

Reply:
@lukas-krecan maybe you have better where to put tests for `InMemoryLockProvider` and avoiding maven module cycles?

Reply:
Yes, I did this [cleanup](https://github.com/lukas-krecan/ShedLock/pull/810). I had to drop the KeepAlive test, but it was disabled anyways